docs/mkdocs/docs/aws.md

```diff
@@ -0,0 +1,73 @@
+# Getting Started with AWS S3
+
+There are detailed guides on the AWS website on S3 configuration.  You can start with the [Creating your first bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/creating-bucket.html) page.
+
+Best practices with AWS S3 can depend on your situation.  If you're looking to try ArcticDB on AWS S3 then a reasonably simple approach is the following.
+
+### 1. Create an IAM user
+
+Best practice is to manage and access S3 buckets with a Non-root IAM account in AWS.   Ideally create a new account just for trialling ArcticDB.
+
+You can create an IAM account at [IAM > Users](https://console.aws.amazon.com/iamv2/home#/users).
+- Click `Add users`.
+- Come up with a username, e.g. `arcticdbtrial`, then click `Next`.
+- Select `Attach policies directly` and attach the `AmazonS3FullAccess` policy to that account, then click `Next`.
+- Click `Create user`.
+
+### 2. Create the access key
+
+- Click on the new user in the [IAM > Users](https://console.aws.amazon.com/iamv2/home#/users) table.
+- Go to `Security credentials` > `Access keys` and click `Create access Key`
+- Select the `Local code` option, ArcticDB is the local code.
+- Check `I understand...` and click `Next`.
+- Click `Create access Key`.
+- Record the Access key and Secret access key somewhere securely.
+
+### 3. Create the bucket
+
+The rest of the steps can be performed using commands on your client machine.  Install the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) if needed.
+
+Configure your client machine to use the new account with the Access key and Secret access key.
+You will also need to select an [AWS region](https://docs.aws.amazon.com/general/latest/gr/rande.html).
+Selecting a region that is local to your ArcticDB client will improve performance.
+```
+aws configure
```

Comment:
Screenshot of how to configure? Like the below I assume?

```
PS C:\Users\alexb> aws configure
AWS Access Key ID [None]: <REDACTED>
AWS Secret Access Key [None]: <REDACTED>
Default region name [None]: eu-west-2
Default output format [None]:
```

Reply:
done inline instead